src/coreclr/vm/jitinterface.cpp

```diff
@@ -1636,6 +1645,60 @@ uint32_t CEEInfo::getThreadLocalFieldInfo (CORINFO_FIELD_HANDLE  field, bool isG
     return typeIndex;
 }
 
+#ifndef _MSC_VER
+
+extern "C" void* JIT_GetThreadStaticsBaseOffset();
```

Comment:
The name should be updated to match what it returns. It does not return offset.

Also, the `JIT_` prefix is used for JIT helpers (methods that are called from JITed code and have `CORINFO_HELP_...` id). This is not a JIT helper. It should not have `JIT_` prefix.

Reply:
got it. I will rename it. I realized that even `jithelpers*.S` files contains the helper. Is there any preference on where this methods should be defined?